### DISCUSSION TITLE
Left click opens Editor Preferences/SpaceMouse Plugin settings

### DISCUSSION THREAD

**Discussion by itognet** [microdee/OpenUnrealSpaceMouse#84] [Q&A] [answered]
I have the SpaceMouse Compact with 2 buttons. 
Each time I left click the UE opens the Editor Preferences / SpaceMouse Plugin settings.

I havnt figured a way to turn it off. Meanwhile I found a hack, placing the Editor Preferences as a tab next to Message Log, so Editor Preferences is always visible so it dont pop up.

Is there a way to disable this behavior ?

**Comment by microdee** [comment, marked as answer]
yes, it's also mentioned in the [README.md](https://github.com/microdee/UE4-SpaceMouse/blob/master/README.md#config--usage), most importantly this two preferences in `Project Settings -> Plugins -> SpaceMouse`
![image](https://user-images.githubusercontent.com/1009962/204131895-fc2f6c08-556f-45d9-a0d5-de7d791dbd51.png)

**Reply by itognet** [reply]
Actually pretty clever to have the SpaceMouse preferences on left click, so users know where to change the settings.
It worked. Sorry for not being soo clever here ;)
